datalad_debian/bootstrap_builder.py

```diff
@@ -28,15 +28,44 @@
 @build_doc
 class BootstrapBuilder(Interface):
     """Bootstrap a build environment
+
+    This command bootstraps a (containerized) build environment, such as a
+    Singularity container, based on an existing builder configuration, such as
+    a Singularity recipe.
```

Comment:
For for "such as" statements, I recommend using parentheses, so that it's clear that they're descriptors rather than relevant for sentence structure.

Reply:
thx!